CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ This is similar to [Angular's CHANGELOG.md](https://github.com/angular/angular/b
 - Add support of [DASpecification](http://eclipse.github.io/winery/user/DASpecification)
 - Add support of configuration of git autocommit: `repository.git.autocommit=true` in `winery.properties`
 - **BREAKING**: in the tosca model `SourceElement` and `TargetElement` are combined into `SourceOrTargetElement` due to serialization issues with JSON
+- Added Backend endpoint for checking Consistency of the Repository in use right now. In Winery-Ui und Adminisistration -> Repository a new Button has been implemented calling the REST Endpoint.
```

Comment:
The nouns (backend, consistency) should be written in lower case. See https://en.wikipedia.org/wiki/Capitalization_in_English.